### DISCUSSION TITLE
Explaination on how the text editor wotk

### DISCUSSION THREAD

**Discussion by DoublEffe** [AppFlowy-IO/AppFlowy#3962] [General help wanted] [answered]
I don't undertand how is it possible multiple field where you can write, how the text editor works

**Comment by annieappflowy** [comment, marked as answer]
Have you checked out [blog.appflowy.io](https://blog.appflowy.io/)?

**Comment by rajarshikhatua** [comment]
yoo, why [this](https://docs.appflowy.io/docs/documentation/appflowy-editor) page is still empty, can't understand a single thing about editor, which is the main thing about the app

**Reply by annieappflowy** [reply]
https://blog.appflowy.io/how-we-built-a-highly-customizable-rich-text-editor-for-flutter/
https://blog.appflowy.io/demystifying-appflowy-editors-codebase/
